src/Plugin/HelpSection/ProfileHelpSection.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Drupal\stanford_profile\Plugin\HelpSection;
+
+use Drupal\help\Plugin\HelpSection\HelpSectionPluginBase;
+
+/**
+ * Provides the module topics list section for the help page.
+ *
+ * @HelpSection(
+ *   id = "profile_help",
+ *   title = @Translation("Getting Started"),
+ *   description = @Translation("What do you need help with today?"),
+ *   weight = -999
+ * )
+ */
+class ProfileHelpSection extends HelpSectionPluginBase {
+
+  use ProfileHelpTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function listTopics() {
+    return [
+      $this->getUserGuide(),
+      $this->getAssistance(),
+      $this->getLaunchWebsite(),
+      $this->getFeedback(),
+      $this->getPolicies(),
+      $this->getContact(),
+    ];
+  }
+
+  /**
+   * Get the user guide help text.
+   *
+   * @return array
+   *   Markup render array.
+   */
+  protected function getUserGuide() {
+    $help = '<h3>' . $this->t('User Guide') . '</h3>';
+    $help .= '<p>' . $this->t('Everything you need to know about how to use, maintain, and launch your D8 Website.') . '</p>';
+    // TODO: Update link when user guide is available.
```

Comment:
```suggestion
    // @TODO: Update link when user guide is available.
```

Make it a real TODO so it can be found later as this is a real TODO and not a cop-out on writing code TODO.

Reply:
Actually, we have a url.

Reply:
oh my ticket in jira didnt have the url. cool!